package.json

```diff
@@ -23,8 +23,6 @@
   },
   "homepage": "https://github.com/koreancanadiansoccer/kcsaweb#readme",
   "devDependencies": {
-    "@types/bcrypt": "^5.0.0",
-    "@types/bcryptjs": "^2.4.2",
```

Comment:
why is this removed?

Reply:
They were imports that weren't being used

Reply:
added them back in